pipeline_dp/dataset_histograms/histogram_error_estimator.py

```diff
@@ -94,13 +94,13 @@ def _get_ratio_dropped(self, ratios_dropped: Sequence[Tuple[int, float]],
         if bound > ratios_dropped[-1][0]:
             return 0
         index = bisect.bisect_left(ratios_dropped, (bound, 0))
-        if ratios_dropped[index] == bound:
+        if ratios_dropped[index][0] == bound:
             return ratios_dropped[index][1]
 
-        x1, y1 = ratios_dropped[index]
-        x2, y2 = ratios_dropped[index + 1]
+        x1, y1 = ratios_dropped[index - 1]
```

Comment:
thanks, good point. I've added a comment and made a small change to ensure that `bound > 0`. 

No we don't need to check, since `ratio_dropped` start from `0`, but here `bound > 0`